vtex-apps/b2b-organizations
# Feat/warning holding organization

#### What problem is this solving?
The idea is to allow merchants to search for the ID and Trade Name fields on the screen that lists organizations in the admin (admin/b2b-organizations/organizations#/organizations).  This implementation has already been done, however, if we publish without preparation, we will prevent accounts from working until they adjust some settings. This activity aims to prepare the accounts for this change.


The change is motivated by the need to notify the user about searching the organization listing. Notifying the user that it is possible to enable and disable the tradeName for the search

#### How to test it?
To do this, we must instruct the merchant to check whether it is possible to search by organization ID and trade name (we recommend reading this page to understand more about the subject). By default, the ID field is usually already indexed. However, the trade name field does not. You for changing this setting via Advanced Settings in Masterdata
Data Structure -> Data Entities -> Edit Customer (CL) -> Procurar por Trade Name e clicar em editar. 

1. go to the organization listing page in the admin
2. see warning alert
3. click on the button and link




[Workspace](Link goes here!)

#### Screenshots or example usage:
![print](https://github.com/user-attachments/assets/b7bb3d39-7f06-4b8f-8bdc-39563c75f9b4)
[Uploading Screen Recording 2024-10-14 at 15.57.05.zip…]()



#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
